src/main/java/de/retest/recheck/report/SuiteReplayResult.java

```diff
@@ -45,6 +45,9 @@
 	@XmlElement
 	private final GroundState replaySutVersion;
 
+	@XmlElement
+	private final ExecutableSuite suite;
```

Comment:
Ideally, we don't have to change persisted classes. Also adding this field might affect the report size. Is there a way around this?